frontend/src/modules/machinery/components/Machinery.tsx

```diff
@@ -123,6 +123,7 @@ export default class Machinery extends React.Component<Props, State> {
                             return (
                                 <Translation
                                     index={index}
+                                    entity={machinery.entity}
```

Comment:
We need to pass the actual value of entity to `Translation`. This is because a search string will not have an entity value. 

We then later check within the `useCopyMachineryTranslation` hook to see if `entity` has a value and if it is `null` we know it is a search string and we add the term to the editor instead of replacing the editor content.